docs/user-guide/simulation.rst

```diff
@@ -1,33 +1,71 @@
 .. currentmodule:: cratermaker
 
-
 .. image:: ../_static/full_simulation.png
     :alt: Simulation
     :align: center
     :width: 300px
 
 .. _ug-simulation:
 
-##########
 Simulation
-##########
+==========
+
+The :class:`Simulation` class manages the process of evolving cratered planetary surfaces through time. It supports configuring a target body, choosing a production function, and generating craters through emplacement or time evolution.
+
+To run a simulation, use the :class:`cratermaker.Simulation` constructor and call :meth:`Simulation.run` or :meth:`Simulation.emplace`.
+
+.. ipython:: python
+    :okwarning:
+
+    from cratermaker import Simulation
+    sim = Simulation(target="Moon")
+    sim.run(age=100)
+
+    print(f"Craters emplaced: {len(sim.true_crater_list)}")
+
+.. _ug-simulation-examples:
+
+Examples
+--------
+
+Below are examples of how to use the Simulation class.
+
+**Example 1: Run a short simulation on Mars**
+
+.. ipython:: python
+    :okwarning:
+
+    from cratermaker import Simulation
+    sim = Simulation(target="Mars")
+    sim.run(age=500)
+    print(f"Crater count: {len(sim.true_crater_list)}")
 
+**Example 2: Simulate default Moon configuration**
 
+.. ipython:: python
+    :okwarning:
 
-The Simulation class is the main orchestration tool used to manage and run landscape evolution simulations.
+    from cratermaker import Simulation
+    sim = Simulation()
+    sim.run(age=1000)
+    print(f"Crater count: {len(sim.true_crater_list)}")
```

Comment:
These simulations take quite a bit of time and they don't actually show much in the way of output. Rather than having examples like this, i would suggest writing up a just a summary of all arguments (e.g. any arguments that can be passed to any of the components can be passed to Simulation() and it will instantiate that component with those arguments). 

Then I can work on some better examples for the gallery.